src/ResourceManager/Common/Commands.ResourceManager.Common/Properties/Resources.resx

```diff
@@ -196,6 +205,9 @@ Use the Enable-AzureRmDataCollection cmdlet to turn the feature On. The cmdlet c
   <data name="TenantNotFound" xml:space="preserve">
     <value>Tenant '{0}' was not found. Please verify that your account has access to this tenant.</value>
   </data>
+  <data name="TimeOutForProviderList" xml:space="preserve">
+    <value>Provider.List() timed out</value>
```

Comment:
If this is a dev message, that is fine, if this is met for custoemrs, we need to give a description that tells them what to do next, and describes the failure not in temrs of underlyign calls (i.e. lisiting locatiosn failed, please try invkoking the cmdlet again"